src/Cell.js

```diff
@@ -99,7 +99,7 @@ const Cell = React.createClass({
   onDragHandleDoubleClick(e) {
     e.stopPropagation();
     let meta = this.props.cellMetaData;
-    if (meta != null && typeof(meta.onDragHandleDoubleClick != null) === 'function') {
+    if (meta != null && typeof(meta.onDragHandleDoubleClick) === 'function') {
```

Comment:
we should also check if `meta.onDragHandleDoubleClick` is null or undefined (by doing `..&& meta.onDragHandleDoubleClick && ..`